contracts/ERC20Decrypto.sol

```diff
@@ -395,6 +484,17 @@ contract ERC20Decrypto is
         _burn(_msgSender(), amount);
     }
 
+    function burnBatch(address[] memory accounts, uint256[] memory amounts)
```

Comment:
I think we may want to remove this.
the owner may only be able to burn tokens that he controls, otherwise they may burn tokens that are not under their control

Reply:
ok, i will fix it